app/models.py

```diff
@@ -65,6 +65,8 @@
 DELIVERY_STATUS_CALLBACK_TYPE = "delivery_status"
 COMPLAINT_CALLBACK_TYPE = "complaint"
 SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE]
+DEFAULT_SMS_ANNUAL_LIMIT = 25000
+DEFAULT_EMAIL_ANNUAL_LIMIT = 10000000
```

Comment:
Do we need these here AND in `config.py`?

Reply:
Sadly yes, as far as I know we aren't able to access the Flask app's environment from `models.py`.

```python
email_annual_limit = db.Column(db.BigInteger, nullable=False, default=current_app.config["SERVICE_ANNUAL_EMAIL_LIMIT"])
```
Results in:
```python
File "/home/vscode/.cache/pypoetry/virtualenvs/notification-api-PZspUUhw-py3.10/lib/python3.10/site-packages/werkzeug/local.py", line 519, in _get_current_object
    raise RuntimeError(unbound_message) from None
RuntimeError: Working outside of application context.
```

Technically we could do: 
```python
from app.config import Config
config = Config()
email_annual_limit = db.Column(db.BigInteger, nullable=False, default=config.SERVICE_ANNUAL_EMAIL_LIMIT)
```

But only for keys that aren't overridden in different environments. It could be useful to separate constant-like configs into their own class so we can reference them outside of the app context.